x-pack/plugins/security_solution/public/management/services/policies/test_mock_utilts.ts

```diff
@@ -7,19 +7,26 @@
 import { FleetPackagePolicyGenerator } from '../../../../common/endpoint/data_generators/fleet_package_policy_generator';
 import { GetPolicyListResponse } from '../../pages/policy/types';
 
-export const sendGetEndpointSpecificPackagePoliciesMock =
-  async (): Promise<GetPolicyListResponse> => {
-    const generator = new FleetPackagePolicyGenerator();
-    const items = Array.from({ length: 5 }, (_, index) => {
-      const policy = generator.generateEndpointPackagePolicy();
-      policy.name += ` ${index}`;
-      return policy;
-    });
+export const sendGetEndpointSpecificPackagePoliciesMock = async (
+  {
+    page,
+    perPage,
+  }: {
+    page: number;
+    perPage: number;
+  } = { page: 1, perPage: 20 }
+): Promise<GetPolicyListResponse> => {
+  const generator = new FleetPackagePolicyGenerator();
+  const items = Array.from({ length: 5 }, (_, index) => {
```

Comment:
What do you think about having this length dynamic and let user define this length when using this method or set it to 5 by default?

Reply:
yeah that doesn't sound like a bad idea, although i guess beyond pagination changes, are there other reasons to be able to change the size?